wmn-data.json

```diff
@@ -3767,6 +3805,16 @@
         "known" : ["anuel", "esteban"],
         "cat" : "xx NSFW xx"
        },
+       {
+        "name" : "myWishBoard",
+        "uri_check" : "https://mywishboard.com/@{account}",
+        "e_code" : 200,
+        "e_string" : "class=\"MwbUserHeader\"",
+        "m_string" : "class=\"MwbError\"",
+        "m_code" : 404,
+        "known" : ["ke7_2024", "jpsol"],

```

Comment:
Please check to see if `jpsol` works for you. I get an error. The other account works.